cmd/ww/server.go

```diff
@@ -21,12 +21,12 @@ import (
 	"time"
 
 	"github.com/NYTimes/gziphandler"
+	"github.com/SpiderOak/webwormhole/wormhole"
 	webrtc "github.com/pion/webrtc/v3"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"golang.org/x/crypto/acme/autocert"
 	"nhooyr.io/websocket"
-	"webwormhole.io/wormhole"
```

Comment:
Why don't we use the replace keyword on the flow repo instead? https://stackoverflow.com/questions/14323872/using-forked-package-import-in-go

Reply:
Good idea.

Reply:
I meant to add the `replace` on the `flow` repo, not in this one. But am not sure exactly what's needed here.

What we need in FLOW-1966 is to run the docker image that can be built by this fork in `tr`. As discussed in the meetings, If we need to use git submodules then I believe that's a-ok too.